dietpi/dietpi-software

```diff
@@ -8795,6 +8795,9 @@ _EOF_
 			# Update APT package status
 			G_EXEC /usr/share/rpimonitor/scripts/updatePackagesStatus.pl
 
+			# Fix issue to display CPU Temperature correctly https://github.com/XavierBerger/RPi-Monitor/issues/374
+			G_CONFIG_INJECT 'dynamic.1.postprocess=' 'dynamic.1.postprocess=$1/1000' /etc/rpimonitor/template/temperature.conf
```

Comment:
To mute shellcheck, since obviously the `$1` is meant literally and not as positional parameter to expand, you can add the following line above `G_CONFIG_INJECT`:
```sh
# shellcheck disable=SC2016
```
Such makes shellcheck skip this rule for the next command/statement.

Reply:
will do, juts playing with it. Still need to find out how to round the value to two digits 🤔